docs/parameters.rst

```diff
@@ -48,9 +48,9 @@ different behavior and some are supported out of the box:
 
 ``bool`` / :data:`click.BOOL`:
     A parameter that accepts boolean values.  This is automatically used
-    for boolean flags.  If used with string values ``1``, ``yes``, ``y``, ``t``
-    and ``true`` convert to `True` and ``0``, ``no``, ``n``, ``f`` and ``false``
-    convert to `False`.
+    for boolean flags.  If used with string values, ``1``, ``yes``, ``y``,
+    ``t``, ``true``, and ``on`` convert to `True`, and ``0``, ``no``, ``n``,
+    ``f``, ``false``, and ``off`` convert to `False`.
```

Comment:
```suggestion
    ``f``, ``false``, and ``off`` the value is `False`.
```

Reply:
That's bad grammar.

Reply:
Seems correct to me, but I went with a different rewrite.